Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -41,21 +41,37 @@ public class BTShopperInsightsClient {
             let postParameters = BTEligiblePaymentsRequest(
                 email: request.email,
                 phone: request.phone,
-                merchantID: "TODO-merchant-id-type"
+                merchantID: "MXSJ4F5BADVNS"
             )
             
             do {
-                let (_, _) = try await apiClient.post("/v2/payments/find-eligible-methods", parameters: postParameters, httpType: .payPalAPI)
-                
-                // TODO: - Handle API Response. DTBTSDK-3388
-                let result = BTShopperInsightsResult()
+                let (json,_) = try await apiClient.post("/v2/payments/find-eligible-methods", parameters: postParameters, httpType: .payPalAPI)
+                let eligibleMethodsJSON: BTJSON = json?["eligible_methods"] ?? BTJSON()
```

Comment:
Also, similar to my comment [here](https://github.com/braintree/braintree_android/pull/894/files#r1471496304) in Android, we should align on what we do when the API response doesn't error (200), but the response body isn't what we expect. 

I think we should surface a new merchant error in that case, similar to [BTSEPADirectDebitError.noBodyReturned](https://github.com/braintree/braintree_ios/blob/075220827e2df164c8230a8ed43b81dba7a402d3/Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitError.swift#L24-L25). We could even add more granular error cases if we see fit